cli/src/lib/libDefs.js

```diff
@@ -570,18 +570,26 @@ function packageNameMatch(a: string, b: string): boolean {
 }
 
 function satisfiesSemver(pkgSemver: string, defVersion: string): boolean {
```

Comment:
I'm concerned about this function being used in contexts where the assumed-`^` isn't intended.
Maybe we should update the name of the function to indicate that it's a non-standard semver comparison?

Maybe something like `pkgVerSatisfiesLibDefVer`? (not tied to that name necessary)